### ISSUE TITLE
hash support for the domain names

### ISSUE THREAD

**Issue by crisc2000** [kafene/history-whitelist-webextension#9]
Could we had some support for the domain list names to be hashed? Because a computer guest can enter in the extension option and easily see what kind of domains you are trying to hide.
At least some kind of base64 encode that is native supported by JavaScript https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding

for example the supported list should be something like this:
`google.com, bWFpbC55YWhvby5jb20, facebook.com`
both plain and base64 support

**Comment by agharbeia** [comment]
I don't think this is the right layer in the system to do this. It's better to have a password on your computer account and encrypted storage.
Besides, Base64 is reversible as to be non-effective.

**Comment by crisc2000** [comment]
"password" -  no, i don't want a password, why typing the same letters 10.000 times over the years, when there is a more simple method to hide those domain names.
"base64" - no, i don't want to protect that against experts, only from the rest of 99% of the peoples. base64 is enough for me and very easy to implement. i could do this modification myself but since Firefox 43 extensions must be signed to work.

**Comment by agharbeia** [comment]
What I was trying to say in my previous comment is that the protection should be on your operating system account, rather than on this level of the browser/extension.

Moreover, things like Base64 are so useless as to not be worth the effort.

I'm not the developer of this extension, however, so it's not up to me decide.

**Comment by crisc2000** [comment]
I have submitted a patch for this request.
The patch was tested and all it's working fine (in blacklist mode).
